domains/raafey.json

```diff
@@ -4,6 +4,10 @@
     "email": "raafeyraza1@gmail.com"
   },
   "record": {
-    "CNAME": "raafey.great-site.net"
+
+    "CNAME": "raafey.great-site.net",
+    "_acme-challenge.raafey.is-a.dev": {
+      "Destination": "dc6fbbae-3aba-4a5d-9741-4b7a386b9151.acmedns.infinityfree.net"
+    }
```

Comment:
you'd need to make _acme-challenge.raafey.is-a.dev in a .json file (without the is-a.dev part)